lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -33,8 +33,8 @@ class ExampleWording < Cop
         MSG_SHOULD = 'Do not use should when describing your tests.'.freeze
         MSG_IT     = "Do not repeat 'it' when describing your tests.".freeze
 
-        SHOULD_PREFIX = 'should'.freeze
-        IT_PREFIX     = 'it '.freeze
+        SHOULD_PREFIX = /\Ashould(?:n't|not)?\b/i
```

Comment:
By using a boundary character instead of a space it will detect single word cases like `it 'shouldn't' {}`.